datautils.py

```diff
@@ -160,41 +110,44 @@ def get_loaders(name, custom_data_path=None, nsamples=128, seed=0, seqlen=2048,
         train_loader (torch.utils.data.DataLoader or iterable): DataLoader for the training dataset.
         test_loader (torch.utils.data.DataLoader or iterable): DataLoader for the test dataset.
     """
-
     set_seed(seed)
 
-    if custom_data_path:
-        dataloader = torch.load(custom_data_path)[: nsamples]
-        return dataloader, None
-
-    assert name != "custom"
+    # handling pre-tokenized datasets
+    if name.lower() == "pajama":
```

Comment:
In prior version one could test performance of their own data...

Reply:
it is still possible.  If the argument value is neither of the presets, it is assumed to be a path to custom data (see line 144). Yet those who want to use pajama or refinedweb can do it easier now.